frame/support/procedural/src/storage/genesis_config/mod.rs

```diff
@@ -104,10 +104,9 @@ fn impl_build_storage(
 		let name = syn::Ident::new(DEFAULT_INSTANCE_NAME, Span::call_site());
 		quote!( #name )
 	});
-	let inherent_instance_bound = def.optional_instance_bound.clone().unwrap_or_else(|| {
-		let bound = syn::Ident::new(DEFAULT_INSTANTIABLE_TRAIT_NAME, Span::call_site());
-		quote!( #inherent_instance: #bound )
-	});
+	let inherent_instance_bound = quote!(
+		#inherent_instance: #scrate::traits::Instance
+	);
```

Comment:
here we use frame_support::traits::Instance directly instead of the internally created instance trait.